ansible.cfg

```diff
@@ -14,5 +14,6 @@ fact_caching_timeout = 0
 inventory = inventory.yml
 pipelining = True
 any_errors_fatal = True
+jinja2_native = True
```

Comment:
`Description: This option preserves variable types during template operations.`

What's the goal here?

Reply:
(ok, it's in the commit message itself)

NOTE: jinja2_native was set to True to avoid Openshift errors like this:
```
spec.numProcesses: Invalid value: "string":
spec.numProcesses in body must be of type integer
```

Reply:
sorry, it's described in one of the commit messages:
https://github.com/openstack-k8s-operators/ci-framework/pull/1185/commits/44904570af9e7ea10c203a88b2c2fd0edd6206b7